client/settings/express-checkout/woopay-item.tsx

```diff
@@ -40,152 +39,117 @@ const WooPayExpressCheckoutItem = (): React.ReactElement => {
 		featureFlags: { woopay: isWooPayFeatureFlagEnabled },
 	} = useContext( WCPaySettingsContext );
 
+	if ( ! isWooPayFeatureFlagEnabled ) {
+		return null;
+	}
+
 	return (
-		<>
-			{ isWooPayFeatureFlagEnabled && (
```

Comment:
I'd recommend to hide whitespaces while reviewing these changes: https://github.com/Automattic/woocommerce-payments/pull/10777/files?w=1

The main changes are:
- Moved the `isWooPayFeatureFlagEnabled` above, for an early return
- Removed the `HoverTooltip` component, replaced it with an `InlineNotice`